### PR TITLE
Add explanations for input fields

### PR DESCRIPTION
**Pull request by rensw90**
This adds explanations to the input fields if provided. Right now only titles and meta's have the an explanation.

To test this:
- Go to step 9 'title settings'
- Check if explanations are shown below the input field for the title and the choice for the title separator.

Fixes https://github.com/Yoast/wordpress-seo/issues/5571

### CODE REVIEW

**Comment by rensw90** [comment]
Fixes https://github.com/Yoast/wordpress-seo/issues/5571

File: composites/OnboardingWizard/components/Choice.js [hunk 1]
```diff
@@ -4,6 +4,23 @@ import Label from "../../../forms/Label";
...
 
 /**
+ * Gets the explanation from the properties.
+ *
+ * @param {string} explanation The explanation text.
+ *
+ * @returns {JSX.Element|null} Returns a paragraph element containing the explanation
+ *                              text if present.
+ */
+var getExplanation = function (explanation) {
```
Comment (andizer) [inline composites/OnboardingWizard/components/Choice.js]: Please move this in a separate file.

File: forms/composites/Textfield.js [hunk 2]
```diff
@@ -36,28 +35,53 @@ class Textfield extends React.Component {
 	}
 
 	/**
+	 * Gets the explanation from the properties.
+	 *
+	 * @returns {JSX.Element|null} Returns a paragraph element containing the explanation
+	 *                               text if present.
+	 */
+	getExplanation() {
```
Comment (andizer) [inline forms/composites/Textfield.js]: Move this in a separate file.

**Comment by andizer** [comment]
CR done.

**Comment by andizer** [comment]
Acceptance done.

**Pull request merged**